lib/core/seleniumRunner.js

```diff
@@ -484,13 +513,38 @@ class SeleniumRunner {
   async runScriptInCategory(category, isAsync) {
     const scriptNames = Object.keys(category);
     const results = {};
+    let requires = [];
+
     for (let scriptName of scriptNames) {
       try {
-        const script = category[scriptName];
+        let script = category[scriptName].content;
+        /*
+         * Assume that if the string in content is null
+         * the function is not null and we want to run
+         * the function.
+         */
+        if (script === null) {
+          assert.ok(
+            category[scriptName].function !== null,
+            'OOPS: Function and script are both null.'
+          );
+          /*
```

Comment:
The style in browsertime is to use `/* */` for JS doc (-ish) things, and `//` for inline comments.  AFAICT.